internal/provider/resource_cloudflare_access_application_test.go

```diff
@@ -255,6 +255,30 @@ func TestAccCloudflareAccessApplication_WithADefinedIdps(t *testing.T) {
 	})
 }
 
+func TestAccCloudflareAccessApplication_WithMultipleIdpsReordered(t *testing.T) {
+	rnd := generateRandomResourceName()
+	idp1 := generateRandomResourceName()
+	idp2 := generateRandomResourceName()
+
+	resource.Test(t, resource.TestCase{
+		PreCheck: func() {
+			testAccPreCheck(t)
+			testAccPreCheck(t)
```

Comment:
I don't know why it is called twice, but since other tests do so, I did so here as well.

Reply:
i've cleaned this up; it shouldn't be duplicated.